GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -216,8 +216,8 @@ DiffBranchStatus GetDiffStatus(GitItemStatus f, bool atoBDiff)
                 secondRev: fileStatusDescs[0].SecondRev,
                 summary: fileStatusDescs[0].Summary,
                 statuses: allAToB,
-                baseA: fileStatusDescs[0].BaseA,
-                baseB: fileStatusDescs[0].BaseB);
+                baseA: baseA,
+                baseB: baseB);
```

Comment:
This is not wrong. But I find the initial version clearer: "just update statuses".

Reply:
The base for the ranges are not set before, base is only used for range-diff. This compares just slices of the branches with a common base.
(could be set on line 107 but more complicated)